src/rpc/handlers/AccountNFTs.cpp

```diff
@@ -89,6 +98,25 @@ AccountNFTsHandler::process(AccountNFTsHandler::Input input, Context const& ctx)
 
         for (auto const& nft : arr) {
             auto const nftokenID = nft[ripple::sfNFTokenID];
+            auto const maskedNftokenID = nftokenID & ripple::nft::pageMask;
+
+            if (!pastMarker) {
+                if (maskedNftokenID < maskedMarker)
+                    continue;
+
+                if (maskedNftokenID == maskedMarker && nftokenID < pageKey)
+                    continue;
+
+                if (nftokenID == pageKey) {
```

Comment:
i understood that this part of code came from rippled's code. However, Clio's implementation is slightly different.
In rippled, the marker is the nft id, however in Clio, the marker is page id. 
So it doesn't make sense to compare nfttokenId with pageKey here. 
The 'limit' works differently. for rippled, limit is the number of nftokens, for clio is the page number. According to the document , It should be page number.